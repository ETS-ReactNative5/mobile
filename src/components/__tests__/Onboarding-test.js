import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Onboarding from '../Onboarding'

jest.mock('ImageBackground')
jest.useFakeTimers();

it('renders', () => {
  const tree = renderer.create(
    <Onboarding />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
