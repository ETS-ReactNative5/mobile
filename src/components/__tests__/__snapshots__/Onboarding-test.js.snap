exports[`test renders 1`] = `
<Image
  source={1}
  style={undefined}>
  <View
    style={undefined}>
    <Image
      source={1}
      style={undefined} />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          undefined,
          Array [
            undefined,
          ],
        ]
      }>
      The Zooniverse enables everyone to take part in real cutting edge research in many fields across the sciences, humanities, and more.
    </Text>
  </View>
  <View
    style={undefined}>
    <View
      style={
        Object {
          "opacity": 0,
        }
      }>
      <TouchableOpacity
        activeOpacity={0.5}
        disabled={false}
        onPress={[Function]}
        style={
          Array [
            undefined,
            Array [
              undefined,
            ],
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              undefined,
              Array [
                undefined,
              ],
            ]
          }>
          Get Started
        </Text>
      </TouchableOpacity>
    </View>
  </View>
</Image>
`;
